From ccea5f27f97a360a517d9f8d70e5a3e9e8c563bf Mon Sep 17 00:00:00 2001
From: Sungyong Cho <to.sungyongcho@gmail.com>
Date: Tue, 10 Jan 2023 18:56:10 +0000
Subject: [PATCH] little-penguin-1 assignment-02

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 460716314fb3..6e615c7562d8 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 2
 SUBLEVEL = 0
-EXTRAVERSION = -rc3
+EXTRAVERSION = -rc3-thor_kernel
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
-- 
2.34.1

